public/app/core/utils/ticks.ts

```diff
@@ -30,127 +30,6 @@ export function getScaledDecimals(decimals: number, tickSize: number) {
   return decimals - Math.floor(Math.log(tickSize) / Math.LN10);
 }
 
-/**
- * Calculate tick size based on min and max values, number of ticks and precision.
- * Implementation from Flot.
- * @param min           Axis minimum
- * @param max           Axis maximum
- * @param noTicks       Number of ticks
- * @param tickDecimals  Tick decimal precision
- */
-export function getFlotTickSize(min: number, max: number, noTicks: number, tickDecimals: number) {
```

Comment:
all this code isn't called from anywhere anymore 🥳

Reply:
Dang, that's a chunk!